server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -82,10 +84,25 @@ private void createDeployScript() throws IOException {
         logger.debug("Creating deploy script.");
         BashScript deployScript = new BashScript(cfnModule.getFileAccess(), FILENAME_DEPLOY);
         deployScript.append(EnvironmentCheck.checkEnvironment("aws"));
+        deployScript.append("source " + FILENAME_UPLOAD + ".sh");
```

Comment:
Doesn't this need a check whether this file is/will be created? Because if there are no files to be uploaded the `file-upload.sh` won't be created and this should throw an error.

Reply:
You're right, I didn't think of this case. I'm going to have to check how ``source file-upload.sh` acts when the file does not exist and probably add a check before that line.